app/scripts/proactive/model/Job.js

```diff
@@ -66,7 +66,7 @@ define(
                         if (StudioApp.isWorkflowOpen()) {
                             that.updateVariable(formValues);
                             var validationData = StudioClient.validate(StudioApp.views.xmlView.generateXml(), StudioApp.models.jobModel);
-                            if (!validationData.valid) {
+                            if (!validationData || !validationData.valid) {
```

Comment:
I don't think this works. !validationData will return true if the validationData object is undefined. And if it's undefined, you will enter this block and will produce an error